ugs-core/src/com/willwinder/universalgcodesender/uielements/jog/StepSizeSpinnerModel.java

```diff
@@ -43,8 +40,11 @@ public Object getNextValue() {
                 stepSize = stepSize / 10;
             }
         }
-        setStepSize(stepSize);
-        return super.getNextValue();
+
+        if( value >= MAX_VALUE) {
+            return 0;
```

Comment:
Looked in to the api docs and i should actually return null. I checked with the old implementation and it did return null as well.
https://docs.oracle.com/javase/7/docs/api/javax/swing/SpinnerNumberModel.html#getNextValue()